nailgun/entities.py

```diff
@@ -138,6 +138,31 @@ def _check_for_value(field_name, field_values):
         )
 
 
+def _get_org_attrs(server_config, organization_label):
```

Comment:
Is it specific reason to have such 'getter' only for Organizations? Or is it first one in a list of such methods?